### ISSUE TITLE
Earthbound

### ISSUE THREAD

**Issue by nstevens26** [Cards-of-Chaos/Game-Design#85]
You target a creature with flying with in range forcing it to make a spell saving throw. On a failed save, vine begin to sprout from the ground, shooting up at the creature wrapping around it, and pulls it down to the ground. The creature is still able to move, but has a flying speed of 0 for 3 turns.

Range: 8 Spaces.
Duration: 3 Turns.

**Comment by nstevens26** [comment]
Should the number of turns be adjusted on this spell?

**Comment by D-D-2** [comment]
I agree changing it from 3 to 2. After some testing it may still be too powerful for single target combats where flying is the main tactical advantage of the baddie, but we'll have to test it
